Misc/NEWS.d/next/Core and Builtins/2017-09-26-16-05-04.bpo-31592.IFBZj9.rst

```diff
@@ -1,2 +1,2 @@
-Fix an assertion failure in case of a bad `unicodedata.normalize()`. Patch
-by Oren Milman.
+Fixed an assertion failure in Python parser in case of a bad `unicodedata.normalize()`.
```

Comment:
Thanks for adding the 'Python parser' part :)

BTW, in https://devguide.python.org/committing/#what-s-new-and-news-entries, the example uses 'Fix ...' (as opposed to 'Fixed ...').
Which phrasing should be used? Or is it unimportant?

Reply:
I prefer 'Fixed ...'. But my English is really bad. I just use old patterns.

About good and bad phrasing read this: https://mail.python.org/pipermail/python-dev/2011-May/111303.html.